AIPscan/Reporter/helpers.py

```diff
@@ -145,3 +146,21 @@ def get_premis_xml_lines(file_object):
         premis_xml_lines = file_object.premis_object.split("\n")
 
     return premis_xml_lines
+
+
+def calculate_paging_window(page, pagination):
+    first_item = ((page - 1) * pagination.per_page) + 1
+    last_item = page * pagination.per_page
+
+    if last_item > pagination.total:
+        last_item = pagination.total
```

Comment:
An alternative to calculate last_item that is more concise by using `min()`:

```suggestion
    last_item = min(page * pagination.per_page, pagination.total)
```

Reply:
Very handy!